cmake/cmake_uninstall.cmake.in

```diff
@@ -0,0 +1,23 @@
+
+if(NOT EXISTS "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
+  message(FATAL_ERROR "Cannot find install manifest: @CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
+endif(NOT EXISTS "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
+
+file(READ "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt" files)
+string(REGEX REPLACE "\n" ";" files "${files}")
```

Comment:
I believe `install_manifest.txt` is generated by CMake, and probably it's not modifiable by us.

Sorry, but I couldn't get what's your concern here. That script replaces `\n` in the string (that contains the list of the installed files with `\n` at the end of each line) to `;` so that CMake interpret it as a list (CMake uses `;` as the delimiter for the list). Please correct me if I'm missing something here.